integrationTests/vm/arwen/arwenvm/mandosConverter/mandosConverterUtils.go

```diff
@@ -0,0 +1,180 @@
+package mandosConverter
+
+import (
+	"fmt"
+	"testing"
+
+	mge "github.com/ElrondNetwork/arwen-wasm-vm/v1_4/mandos-go/elrondgo-exporter"
+	mgutil "github.com/ElrondNetwork/arwen-wasm-vm/v1_4/mandos-go/util"
+
+	"github.com/ElrondNetwork/elrond-go-core/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm"
+	"github.com/ElrondNetwork/elrond-go/state"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+	"github.com/stretchr/testify/require"
+)
+
+// CheckAccounts will verify if mandosAccounts correspond to AccountsAdapter accounts
+func CheckAccounts(t *testing.T, accAdapter state.AccountsAdapter, mandosAccounts []*mge.TestAccount) {
+	for _, mandosAcc := range mandosAccounts {
+		accHandler, err := accAdapter.LoadAccount(mandosAcc.GetAddress())
+		require.Nil(t, err)
+		account := accHandler.(state.UserAccountHandler)
+
+		require.Equal(t, mandosAcc.GetBalance(), account.GetBalance())
+		require.Equal(t, mandosAcc.GetNonce(), account.GetNonce())
+
+		scOwnerAddress := mandosAcc.GetOwner()
+		if len(scOwnerAddress) == 0 {
+			require.Nil(t, account.GetOwnerAddress())
+		} else {
+			require.Equal(t, mandosAcc.GetOwner(), account.GetOwnerAddress())
+		}
+
+		codeHash := account.GetCodeHash()
+		code := accAdapter.GetCode(codeHash)
+		require.Equal(t, len(mandosAcc.GetCode()), len(code))
+
```

Comment:
If the `mandosAcc` contains `CodeMetadata`, it should be asserted here as well against the `CodeMetadata` in `accHandler`.

Reply:
As far as I know, `CodeMetadata` is currently not available in mandos